hub/htypes.py

```diff
@@ -59,6 +59,7 @@
 COMMON_CONFIGS = {
     "sample_compression": None,
     "dtype": None,
+    "max_chunk_size": None,
```

Comment:
default being none here means that it can be adaptive (in case later we change the default max chunk size)